_posts/en/newsletters/2022-09-21-newsletter.md

```diff
@@ -42,7 +42,22 @@ This week's newsletter FIXME
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-<!-- FIXME:bitschmidty -->
+- **Mempool Project launches Lightning Network explorer:**
+  Mempool's open source [Lightning dashboard][mempool lightning] shows aggregate
+  network statistics as well as individual node liquidity and connectivity data.
+
+- **Federation software Fedimint adds Lightning:**
+  In a recent [blog post][blockstream blog fedimint], Blockstream outlines
+  updates to the [Fedimint][] federated Chaumian e-cash project, including
+  Lightning Network support.
+
+- **Bitpay wallet improves RBF support:**
+  Bitpay [improves][bitpay 12051] its [existing][bitpay 11935] support for
+  sending [RBF][topic rbf] transactions.
```

Comment:
The commit is not particularly well-explained, but I have the impression that this is a bug fix. It seems to me that if the RBF function was called on a multi-recipient transaction it would only send to a single recipient in the replacement transaction, and now they support bumping multi-recipien transactions. I'm not sure if that's worth mentioning, but if it is, we might want to give a slightly more specific description?

Reply:
Yeah, that is my take as well. I mostly used the recent fix commit as a way to point out the RBF support that was added previously, but apparently I missed it back then. I pushed a small change to add to this description.

@harding Im cool with you dropping this entry, your call!

Reply:
I'd slightly prefer to keep it in its current edited form.  I think it's ok in this section to give a hat tip to a provider implementing a useful feature without going into details about a specific bug in their implementation.